### ISSUE TITLE
constOffsets seem to be broken (pointing to wrong addresses?)

### ISSUE THREAD

**Issue by Combine12** [WarlockD/GMdsam#4]
It seems that with the last 2 versions of the program, the "constOffsets" has been broken. For example, obj_heart, line 27:
`self.shot= 0/* ([0x00A42578],2) */;`

This offset should lead to a variable that when set to 1, allows the heart to shoot like it can during the mettaton fights, however this doesn't work for some reason.

Even further error can be seen with line 41 of obj_mainchara for example,
`self.cutscene= 0/* ([0x00A4E4DC],2) */;`

This line should (to my understanding) lead to a hexadecimal "00" in the data.win but when I go there it leads to "F8".

I recall the heart offset at least working in previous versions, but I just tested the most recent 2 and both of them had the same result.

**Comment by colinator27** [comment]
I sort of noticed this but wasn't 100% sure if it was actually doing it or if I was being stupid. :P

**Comment by Combine12** [comment]
Looks like your suspicions were right then.
Any ETA on a fix, warlock? It's quite annoying, there's a couple of things I wanted to test with the  "show all constant offsets" feature. I fear the problem may have been a result of the new type system but i'm no expert. 

It appears the addresses are offset by a certain amount for each different code file, for example in obj_heart, all the addresses are offset by +170a6c, while in obj_soundtest it's +2D9988. Very peculiar.

**Comment by WarlockD** [comment]
Sorry during the week I am busy so I don't always check.

Ok I found the problem.  I was using the wrong variable to calculate the offset.  I had forgotten I changed its name to something else.:P  I tested to make sure they are the correct offset by making a debug function that reads the raw data.win and compares it to all the offsets to make sure they are equal.

Anyway, give me a few mins and I will update the release.
